cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -92,8 +92,14 @@
     </property>
 
     <property>
+        <name>user.dir</name>
```

Comment:
Better not to call it `user.dir` as it clash with the standard java system property. Call it something like `local.root.dir` and have the value as `${user.dir}`.